src/components/Footer/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+export default class Footer extends Component {
+  render() {
+    const Footer = styled.form`
```

Comment:
This should not be the form tag, rather div.
Checkout flexboxfroggy.com, it's a nice layout model